commerce_coordinator/apps/lms/views.py

```diff
@@ -4,76 +4,52 @@
 import logging
 from urllib.parse import unquote, urlencode
 
-from django.conf import settings
-from django.http import HttpResponseRedirect, HttpResponseServerError
+from django.http import HttpResponseBadRequest, HttpResponseRedirect
 from edx_rest_framework_extensions.permissions import LoginRedirectIfUnauthenticated
-from rest_framework.parsers import JSONParser
 from rest_framework.status import HTTP_303_SEE_OTHER
 from rest_framework.throttling import UserRateThrottle
 from rest_framework.views import APIView
 
-from ..core.constants import HttpHeadersNames, MediaTypes, QueryParamPrefixes, WaffleFlagNames
-from .filters import OrderCreateRequested
-from .serializers import OrderCreatedSignalInputSerializer
+from commerce_coordinator.apps.core.constants import HttpHeadersNames, MediaTypes
+from commerce_coordinator.apps.lms.filters import PaymentPageRedirectRequested
 
 logger = logging.getLogger(__name__)
 
 
-class OrderCreateView(APIView):
-    """Accept incoming request for creating a basket/order for a user."""
+class PaymentPageRedirectView(APIView):
+    """Accept incoming request for routing users to checkout view."""
     permission_classes = (LoginRedirectIfUnauthenticated,)
     throttle_classes = (UserRateThrottle,)
-    parser_classes = [JSONParser]
 
-    def get(self, request):  # pylint: disable=inconsistent-return-statements
+    def get(self, request):
         """
-        Create orders for an authenticated user.
+        Routes learners to desired checkout view
 
         Args:
             request (django.http.HttpRequest): django.http.HttpRequest
 
-        Query Parameters:
-            sku: Array. An edx.org stock keeping units (SKUs) that the user would like to purchase.
-            coupon_code: (Optional) A coupon code to initially apply to the order.
-
         Returns:
             an HTTP Response, in the form of an error or as an HTTP Redirect.
 
-            - HTTP Redirect (303): Redirection to Payment MFE upon Success
+            - HTTP Redirect (303): Redirection to Commercetools or Legacy checkout upon Success
```

Comment:
Nit: Make more generic so it's independent of pipeline configuration:

```suggestion
            - HTTP Redirect (303): Redirection to correct checkout page upon Success
```